templates/template-app/.gitignore

```diff
@@ -0,0 +1 @@
+build/
```

Comment:
Recommend adding `sdkconfig` and `sdkconfig.old` here.

Reply:
Added in #38